test/e2e/nodemaintenance_test.go

```diff
@@ -90,6 +96,7 @@ func TestNodeMainenance(t *testing.T) {
 	t.Run("NodeMaintenance-group", func(t *testing.T) {
 		t.Run("Cluster", ClusterTest)
 	})
+
```

Comment:
> No leftovers, experimentation, or stray whitespace in PRs that are being reviewed

Reply:
done

Reply:
also did the last initDrainer comment (but can't comment on that)